packages/components/input/src/input.css

```diff
@@ -0,0 +1,37 @@
+/* General........................................................ */
+
+:host,
+:host > div {
+	display: block;
```

Comment:
The div by default is a block-level element, can you check please if this line required?

Reply:
👍